docs/source/install_rocm.rst

```diff
@@ -1,66 +1,28 @@
-[Experimental] Installation Guide for ROCm environemt
-=====================================================
-
-.. contents:: :local:
-
-This is an experimental feature. We recommend only for advanced users to use this.
-
-Recommended Environments
-------------------------
-
-We recommend the following Linux distributions.
-
-* `Ubuntu <https://www.ubuntu.com/>`_ 16.04 / 18.04 LTS (64-bit)
+Using CuPy on AMD GPU (experimental)
+====================================
 
+CuPy has an experimental support for AMD GPU (ROCm).
 
 Requirements
 ------------
 
-You need to have the following components to use CuPy.
+* `AMD GPU supported by ROCm <https://github.com/RadeonOpenCompute/ROCm#Hardware-and-Software-Support>`_
 
-* GPU supported by ROCm (AMD GPUs or NVIDIA GPUs)
-* `ROCm <https://rocm.github.io/install.html>`_
-    * Supported Versions: ROCm 2.6+.
-* `Python <https://python.org/>`_
-* `NumPy <http://www.numpy.org/>`_
+* `ROCm <https://rocmdocs.amd.com/en/latest/index.html>`_: v2.6+
```

Comment:
@leofang is the support limited only to 3.5?

Reply:
Currently, yes. I will create a PR to support 3.5+.

Reply:
I think we should change the 2.6 here for 3.5 right?

Reply:
> I will create a PR to support 3.5+.

See #4110. I think with that quick fix it should be safe to say 3.5.0+ here, although I'll need to find some time to test 3.8.0 too.

Reply:
btw We definitely don't wanna mention ROCm 2.x anymore, as I think a lot of components are incompatible now.